pkg/querier/flamegraph.go

```diff
@@ -45,37 +48,40 @@ func NewFlameGraph(t *tree) *querierv1.FlameGraph {
 		var i int
 		var ok bool
 		name := current.node.name
-		if i, ok = nameLocationCache[name]; !ok {
-			i = len(names)
-			if i == 0 {
-				name = "total"
+		if current.node.total >= minVal || name == "other" {
```

Comment:
Assuming the "other" is not implemented right now (maybe I missed it).

In my opinion we should be transparent, there is "more". But that could also be a follow-up PR.